she_codes_news/news/models.py

```diff
@@ -1,8 +1,14 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
+USER = get_user_model()
 
 class NewsStory(models.Model):
     title = models.CharField(max_length=200)
-    author = models.CharField(max_length=200)
+    # author = models.CharField(max_length=200)
+    author = models.ForeignKey(
+        USER, on_delete=models.CASCADE
+    )
     pub_date = models.DateTimeField()
     content = models.TextField()
+    image_url = models.URLField(max_length=200, default='https://img.freepik.com/premium-photo/astronaut-outer-open-space-planet-earth-stars-provide-background-erforming-space-planet-earth-sunrise-sunset-our-home-iss-elements-this-image-furnished-by-nasa_150455-16829.jpg')
```

Comment:
urlfield defaults to 200 characters, so max_length=200 isn't needed. 
I like the use of a default here though!